tests/callbacks/test_callbacks.py

```diff
@@ -326,3 +327,30 @@ class CurrentTestModel(LightTestOptimizersWithMixedSchedulingMixin,
         'Number of learning rates logged does not match number of lr schedulers'
     assert all([k in ['lr-Adam', 'lr-Adam-1'] for k in lr_logger.lrs.keys()]), \
         'Names of learning rates not set correctly'
+
+
+def test_lr_logger_param_groups(tmpdir):
+    """ Test that learning rates are extracted and logged for single lr scheduler"""
+    tutils.reset_seed()
+
+    class CurrentTestModel(LightTestOptimizerWithParamGroups, LightTrainDataloader, TestModelBase):
```

Comment:
Pls use new `EvalModelTemplate` instead and define custom `configure_optimizers` if needed
see: https://github.com/PyTorchLightning/pytorch-lightning/blob/master/tests/base/eval_model_optimizers.py